docs/build-with-powerloom/snapshotter-node/full-node/getting-started.md

```diff
@@ -0,0 +1,137 @@
+---
+sidebar_position: 2
+---
+# Getting Started
+
+Snapshotter (Full Node) has a complicated architecture with multiple moving parts. The easiest way to get started is by using the Docker-based setup, recommended for most users.
```

Comment:
complex, not complicated.

Reply:
fixed in https://github.com/PowerLoom/docs/pull/8/commits/716a2942de5d0fabe887bd2472655ebabc5d808a